packages/asset-swapper/contracts/src/MStableSampler.sol

```diff
@@ -49,6 +49,7 @@ contract MStableSampler is
         view
```

Comment:
NIT: now that we've lowered the gas schedule we should probably also adjust `DEFAULT_CALL_GAS`

Reply:
This is just for the sampler right? I'm wondering, does this matter to have an "accurate" gas amount here? Or we could just set a big value and forget about it? cc @dekz @merklejerk

Reply:
Yeah, it doesn't have to be accurate, just reasonable because it can eat into our total gas allowance for this call if it reverts. It doesn't have to reflect the swap gas schedule.